app/assets/stylesheets/views/mod-actions.scss

```diff
@@ -527,6 +527,30 @@
           margin-bottom: var(--su-3);
         }
       }
+      #tag-moderation-history {
```

Comment:
(non-blocking): Currently we use the utility based approach for our CSS using crayons, and at a glance it seems like a lot of these can be added as utility classes on our HTML rather than adding custom css.